shoop/admin/views/dashboard.py

```diff
@@ -22,3 +23,7 @@ def get_context_data(self, **kwargs):
             blocks.extend(module.get_dashboard_blocks(request=self.request))
         context["activity"] = get_activity(request=self.request)
         return context
+
+    def get(self, request, *args, **kwargs):
+        send_telemetry(request)
+        return super(DashboardView, self).get(request, *args, **kwargs)
```

Comment:
Not having a separate public "should send" function is a conscious choice.

I don't want it to ever be possible for telemetry to be accidentally sent.